.gitignore

```diff
@@ -1,2 +1,3 @@
 config.js # visor token var config = {visorToken: "tokenishere"};
-build/
\ No newline at end of file
+build/
+.*/
```

Comment:
Have never seen this before. Is this to force folks to manually add any hidden files?